### ISSUE TITLE
Accessibility setting "high contrast" makes buttons hard to recognise

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#38929]
### Steps to reproduce the issue
1. Set "high contrast" in accessibility settings

### Expected result
Something like this: 
![grafik](https://user-images.githubusercontent.com/1035262/194796352-64d3f5a2-f893-41cc-89b4-38f9dc0c9b70.png)

### Actual result
Background is white, that's ok. But without border the elements are difficult to recognize. They need a border. 

![grafik](https://user-images.githubusercontent.com/1035262/194796403-9b020dbf-d923-44d1-90aa-061c45713770.png)

![grafik](https://user-images.githubusercontent.com/1035262/194795825-7fa762d7-6f2c-441c-93e5-eb0ba5e39d14.png)

### Device and system info

* What version of Joomla were you using?
4.2.2, 4.2.3

* What device were you using? (eg, phone, laptop)
laptop

* What web browser were you using? (eg, Chrome, Safari)
firefox, chrome

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by KumarRishabh22** [comment]
Hey I want to work on this issue so, plz assign me

**Comment by chmst** [comment]
Thank you, I assigned you.

**Comment by chmst** [comment]
@KumarRishabh22 how are you performing with this issue? Changes in Joomla css are not as easy ... I know that.

**Comment by ShaileshKumar007** [comment]
Hi @chmst . Actually I'm working on this issue currently. 
I'm able to set the border when the page is in its default setting (i.e. when high contrast is off) but I'm not able to find the css code which is responsible when high contrast is on.
Can you guide me a lil bit?

**Comment by soumyabasuri** [comment]
@chmst  Hey I want to work on this issue so, plz assign me

**Comment by chmst** [comment]
Closing this as obviously no one ever used this setting.
